ethereumj-core/src/main/java/org/ethereum/crypto/ECKey.java

```diff
@@ -345,9 +516,13 @@ public ECPoint getPubKeyPoint() {
      * @throws java.lang.IllegalStateException if the private key bytes are not available.
      */
     public BigInteger getPrivKey() {
-        if (priv == null)
+        if (privKey == null) {
+            throw new MissingPrivateKeyException();
+        } else if (privKey instanceof BCECPrivateKey) {
+            return ((BCECPrivateKey) privKey).getS();
```

Comment:
They are the same.
The Java security interface uses the name `S` for the private key value:
https://docs.oracle.com/javase/8/docs/api/java/security/interfaces/ECPrivateKey.html#getS--
The Bouncy Castle interface uses the name `D` for the private key value:
http://www.bouncycastle.org/docs/docs1.5on/org/bouncycastle/jce/interfaces/ECPrivateKey.html#getD()

But ultimately it’s the same value:
https://github.com/bcgit/bc-java/blob/3d610624e9e87b8abee4998dd899259d8511449c/prov/src/main/java/org/bouncycastle/jcajce/provider/asymmetric/ec/BCECPrivateKey.java#L339-L347


The analogue occurs for the public point:
https://docs.oracle.com/javase/8/docs/api/java/security/interfaces/ECPublicKey.html#getW--
http://www.bouncycastle.org/docs/docs1.5on/org/bouncycastle/jce/interfaces/ECPublicKey.html#getQ()
https://github.com/bcgit/bc-java/blob/3d610624e9e87b8abee4998dd899259d8511449c/prov/src/main/java/org/bouncycastle/jcajce/provider/asymmetric/ec/BCECPublicKey.java#L330-L343
although there the `ECPoint`s are different nominal types, despite again being the same pair of values